packages/frontend/src/features/scheduler/components/SchedulerForm.tsx

```diff
@@ -80,7 +80,7 @@ enum SlackStates {
 const DEFAULT_VALUES = {
     name: '',
     message: '',
-    format: SchedulerFormat.CSV,
+    format: SchedulerFormat.IMAGE,
```

Comment:
We want to change the default? That seems fine with me. Should we also switch the order of these? So the default is on the left still

<img width="317" alt="Screenshot 2024-02-16 at 17 25 24" src="https://github.com/lightdash/lightdash/assets/1864179/da7bb18f-abd0-4655-a80f-86103932d068">

Reply:
ohh, good question. no, I was only supposed to change the default for ALERT,

I push a change. 

good catch.